user/how-to-guides/how-to-open-urls-in-other-qubes.md

```diff
@@ -1,174 +1,105 @@
 ---
 lang: en
 layout: doc
-permalink: /doc/how-to-open-urls-in-other-qubes/
-ref: TBD
```

Comment:
No need to worry about this. There's a script that will assign an appropriate number and add this line automatically.